docs/chapters/architecture/SOTA-system.rst

```diff
@@ -0,0 +1,614 @@
+:orphan:
+
+.. _SOTA-system:
+
+.. toctree::
+    :hidden:
+
+    ../../swf-blueprint/docs/articles/architecture/arch-vertical-configurations.rst
+    ../../swf-blueprint/docs/articles/architecture/arch-glossary.rst
+
+SOTA system
+===========
+
+Introduction and goals
+----------------------
+
+The SOTA System is responsible for distributing and applying updates using
+over-the-air mechanisms so that PELUX-running machines can get new
```

Comment:
I see that you have used this as a sort of "identifyer" later on, but I still feel that I would like you to rewrite this as "machines running PELUX".
Maybe others disagree.